Image Rendering: New setting to control render request concurrency (#23950)
grafana/grafana: docs/sources/installation/configuration.md

If the remote HTTP image renderer service runs on a different server than the Grafana server you may have to configure this to a URL where Grafana is reachable, e.g. http://grafana.domain/.

### concurrent_render_request_limit

> Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server
> and this setting can help protect against that by only allowing certain amount of concurrent request.

Reviewer: ```suggestion
Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server,
which this setting can help protect against by only allowing a certain amount of concurrent requests.
```

> Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server
> and this setting can help protect against that by only allowing certain amount of concurrent request.

Author: ```suggestion
Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server and this setting can help protect against that by only allowing certain amount of concurrent requests.
```
